pages/mi/profiles/priya0030.md

```diff
@@ -0,0 +1,31 @@
+# Priya Dobariya
+
+| Location               | Timezone      | OS                         |
+| ---------------------- | ------------- | -------------------------- |
+| Birmingham, Alabama    | GMT -5        | Windows 11 Home 22631.3447 |
+
+## About Me:
+
+-🎓 Currently pursuing an MS in Computer Science at UAB.
+
+-📱 Mobile Application Developer passionate about creating innovative apps.
+
+-🔄 Loves exploring new ideas and trying different things.
+
+-🎧 Enjoys listening to music.
+
+## Languages and Technologies mostly I use:
+
+  <li> C </li>
+  <li> C++ </li>
+  <li> Java </li>
+  <li> Python </li>
+  <li> Dart </li>
+  <li> Flutter </li>
+  <li> Android Studio </li>
+  <li> Git </li>
```

Comment:
[List is available with markdown syntax](https://www.markdownguide.org/basic-syntax/#lists-1). Please avoid HTML tags here since we are practicing writing markdown in this step.